billing-service/src/main/java/com/petclinic/billing/datalayer/BillRepository.java

```diff
@@ -1,16 +1,21 @@
 package com.petclinic.billing.datalayer;
 
-import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.mongodb.repository.ReactiveMongoRepository;
+import org.springframework.stereotype.Repository;
 import org.springframework.transaction.annotation.Transactional;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
 
-
-import java.util.List;
-import java.util.Optional;
-
-public interface BillRepository extends JpaRepository<Bill, Integer> {
+//Changed from JPA to ReactiveCRUD, makes the repo reactive without changing the entity
```

Comment:
Out of date comments?

Reply:
Yup, ended up changing the repo to Mongo eventually